src/VS.Web.CG.Design/VS.Web.CG.Design.csproj

```diff
@@ -23,27 +24,55 @@
     <ProjectReference Include="$(AnyCpuProjectDirectory)\VS.Web.CG.Design-anycpu.csproj" PrivateAssets="true" ReferenceOutputAssembly="false" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Include="$(OutputPath)netcoreapp2.0\$(AssemblyName).dll" Pack="true" PackagePath="lib\netcoreapp2.0\$(AssemblyName).dll" Visible="false" />
-    <None Include="$(OutputPath)netcoreapp2.0\$(AssemblyName).xml" Pack="true" PackagePath="lib\netcoreapp2.0\$(AssemblyName).xml" Visible="false" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(OS)'=='Windows_NT'">
-    <None Include="$(OutputPath)net461\win7-x64\$(AssemblyName).exe" Pack="true" PackagePath="runtimes\win7-x64\lib\net461\$(AssemblyName).exe" Visible="false" />
-    <None Include="$(OutputPath)net461\win7-x64\$(AssemblyName).xml" Pack="true" PackagePath="runtimes\win7-x64\lib\net461\$(AssemblyName).xml" Visible="false" />
-    <None Include="$(X86ProjectDirectory)\bin\$(Configuration)\net461\win7-x86\$(AssemblyName).exe" Pack="true" PackagePath="runtimes\win7-x86\lib\net461\$(AssemblyName).exe" Visible="false" />
-    <None Include="$(X86ProjectDirectory)\bin\$(Configuration)\net461\win7-x86\$(AssemblyName).xml" Pack="true" PackagePath="runtimes\win7-x86\lib\net461\$(AssemblyName).xml" Visible="false" />
-    <None Include="$(ARMProjectDirectory)\bin\$(Configuration)\net461\win-arm\$(AssemblyName).exe" Pack="true" PackagePath="runtimes\win-arm\lib\net461\$(AssemblyName).exe" Visible="false" />
-    <None Include="$(ARMProjectDirectory)\bin\$(Configuration)\net461\win-arm\$(AssemblyName).xml" Pack="true" PackagePath="runtimes\win-arm\lib\net461\$(AssemblyName).xml" Visible="false" />
-    <None Include="$(ARM64ProjectDirectory)\bin\$(Configuration)\net461\win-arm64\$(AssemblyName).exe" Pack="true" PackagePath="runtimes\win-arm64\lib\net461\$(AssemblyName).exe" Visible="false" />
-    <None Include="$(ARM64ProjectDirectory)\bin\$(Configuration)\net461\win-arm64\$(AssemblyName).xml" Pack="true" PackagePath="runtimes\win-arm64\lib\net461\$(AssemblyName).xml" Visible="false" />
-    <None Include="$(AnyCpuProjectDirectory)\bin\$(Configuration)\net461\$(AssemblyName).exe" Pack="true" PackagePath="lib\net461\$(AssemblyName).exe" Visible="false" />
-    <None Include="$(AnyCpuProjectDirectory)\bin\$(Configuration)\net461\$(AssemblyName).xml" Pack="true" PackagePath="lib\net461\$(AssemblyName).xml" Visible="false" />
-  </ItemGroup>
-
   <ItemGroup>
     <!-- Set to PrivateAssets=All so the package does not appear in the generated nuspec. This prevents issues such as https://github.com/aspnet/Scaffolding/issues/645. -->
     <PackageReference Update="Microsoft.NETCore.App" PrivateAssets="All" />
   </ItemGroup>
 
+  <Target Name="PopulateNuspec" BeforeTargets="GenerateNuspec" DependsOnTargets="Build">
```

Comment:
Try adding ResolveCommitHash to DependsOnTargets

Reply:
The target "ResolveCommitHash" does not exist in the project.

Reply:
Do we still want to add it? It wasn't adding it before and `dotnet pack` used to work. Not sure if we want to break that.

Reply:
No, don't add the DependsOnTargets. But we should still add `commit="$RepositoryComimt$"` to the nuspec

Reply:
That's what I meant. Are we okay with `dotnet pack` not working?

Reply:
Oh got it